frontend/src/libs/components/button/button.tsx

```diff
@@ -5,7 +5,7 @@ import { type ReactMouseEvent, type ValueOf } from '~/libs/types/types.js';
 import { Loader } from '../components.js';
 import styles from './styles.module.scss';
 
-type ButtonVariant = 'primary' | 'outlined' | 'text';
+type ButtonVariant = 'primary' | 'greenOutlined' | 'whiteOutlined' | 'text';
```

Comment:
Let's separate it to `color` prop and keep outlined variant as it was

Reply:
Isn't it better to have two variants of that button, in accordance with our design, I believe, we have only two outlined buttons.
Do we need to get the opportunity to set outlines in different colors?)

Reply:
I thought that mean that main color of button would be green\white